netbout-web/src/test/java/com/netbout/email/EmMessagesITCase.java

```diff
@@ -45,9 +53,25 @@
  * @author Matteo Barbieri (barbieri.matteo@gmail.com)
  * @version $Id$
  */
+@SuppressWarnings("PMD.AvoidInstantiatingObjectsInLoops")
 public final class EmMessagesITCase {
 
     /**
+     * Send it.
+     */
+    private static final String SEND_IT = "send it";
```

Comment:
@pecko I don't see a need for this to be a class variable since it can be a method variable.